websites/S/SOAP2DAY/dist/metadata.json

```diff
@@ -24,7 +25,7 @@
 		"s2dfree.de",
 		"s2dfree.is"
 	],
-	"version": "3.0.0",
+	"version": "3.1.0",
```

Comment:
```suggestion
	"version": "3.0.1",
```

Reply:
Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8